apps/web/src/api/axios.client.ts

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+
+axios.interceptors.response.use(
+  (response) => {
+    return response;
+  },
+  (error) => {
+    if (error.response.status === 401) {
+      localStorage.removeItem('auth_token');
+      localStorage.removeItem('widget_user_auth_token');
+      window.location.href = '/auth/login';
+    }
```

Comment:
This makes an assumption that all unauthorized responses should also de-authenticate the user.  I feel this is not correct and should not be implemented.

Reply:
Tend to aggree here, there was a discussion on Discord about that aswell.